laravel/resources/views/pages/applications/list.blade.php

```diff
@@ -1,15 +1,25 @@
+<?php
+    use Carbon\Carbon;
+?>
 @extends('app')
 
 @section('content')
     <h1>All Applications
-         @if(in_array(Auth::user()->role, ['admin']))
+        @if(in_array(Auth::user()->role, ['admin']))
             <div class="pull-right" style="font-size:0.4em; margin-top: 1.4em;"><a href="/recalcscores">Recalculate Scores</a></div>
         @endif
     </h1>
     @foreach($rounds as $round)
         <hr>
-        <h2> {{ $round->name }} </h2>
-        <div class="scrollable">
+        <h2 class="appTitle"> {{ $round->name }} </h2>
+
+        <!--check to see if the round end_date is a year old from today-->
+
+        @if(Carbon::parse($round->end_date)->diffInDays(Carbon::now())==365)
```

Comment:
Um, wouldn't this only work if the difference in days is exactly 365?

Reply:
interestingly enough it works regardless when using diffInYears(), but i think thats something that could be made more clear as well. I tested both and they work.